packages/flutter_tools/lib/src/linux/build_linux.dart

```diff
@@ -45,6 +46,7 @@ Future<void> buildLinux(
 
   final List<ProjectMigrator> migrators = <ProjectMigrator>[
     CmakeCustomCommandMigration(linuxProject, logger),
+    CmakeNativeAssetsMigration(linuxProject, 'linux', logger),
```

Comment:
The `'linux'` string gets pasted into the Cmake file. The implications of this aren't evident from the call site.

Would it make sense to pass an enum here and have `CmakeNativeAssetsMigration` decide what OS string value should be used when generating the cmake file?

Reply:
I didn't see anything in `project` or `project.parent` to give me access to the OS.